velox/connectors/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ target_link_libraries(velox_connector velox_config velox_vector)
 
 if(${VELOX_ENABLE_HIVE_CONNECTOR})
   add_subdirectory(hive)
+  add_subdirectory(tpch)
```

Comment:
yeah, I left it was a placeholder until we discuss how to support it. Should I create a new generic flag for all connectors VELOX_ENABLE_CONNECTORS, or one specific for the TpchConnector?

Reply:
> should we be enabling the tpch connector only if the hive connector is enabled? They are independent correct?

it would be nice to make TPC-H connector available separately from Hive connector.

Reply:
> Should I create a new generic flag for all connectors VELOX_ENABLE_CONNECTORS, or one specific for the TpchConnector?

A single flag would work, I think. At least until we get a lot more connectors.

Reply:
Created a new flag for now.